workers/event_catcher/event_catcher.rb

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env ruby
+
+require "manageiq-messaging"
+require "manageiq-password"
+require "pathname"
+
+class EventCatcher
+  def initialize(ems_id, hostname, username, password, port, messaging_host, messaging_port, page_size = 20)
+    @ems_id         = ems_id
+    @hostname       = hostname
+    @username       = username
+    @password       = ManageIQ::Password.try_decrypt(password)
```

Comment:
hmmm... from a byoi POV, I think we should avoid depending on ManageIQ password, and just pass it as an env var?

Reply:
We discussed offline but the unit config files are written to `/etc/systemd/system/unit@guid.d/overrides.conf` so passwords should be encrypted at rest

Reply:
Going to work on a utility to decrypt env vars so that calling `ManageIQ::Password` directly isn't necessary